test/test.js

```diff
@@ -29,6 +29,19 @@ describe('url-parse', function () {
     assume(url.hostname).to.be.a('string');
   });
 
+  it('parse when global variable is undefined', function () {
```

Comment:
Maybe change the title to something like "works when the `global` variable is not defined".